knowage-core/src/main/java/it/eng/spagobi/api/common/AbstractDataSetResource.java

```diff
@@ -1048,7 +1048,7 @@ public String getDataSet(String label) {
 		}
 	}
 
-	public Response deleteDataset(String label) {
+	protected Response deleteDataset(String label) {
```

Comment:
Do you think it's possible to use private? If there is no inheritance I would prefer the private modifier.

Reply:
This one can't be private, it is being used in DataSetResource.java (two files, one of them is v2).